group_vars/role_corerouter/imageprofile.yml

```diff
@@ -1,25 +1,43 @@
 ---
 role_corerouter__packages__to_merge:
-  - babeld
+  # monitoring
   - collectd-mod-olsrd
   - collectd-mod-exec
   - collectd-mod-conntrack
   - collectd-mod-snmp
   - snmp-mibs
+  
+  # routing
+  - babeld
   - olsrd
   - olsrd-mod-arprefresh
   - olsrd-mod-dyn-gw
   - olsrd-mod-jsoninfo
   - olsrd-mod-nameservice
   - olsrd-mod-txtinfo
   - olsrd-utils
+  
+  # UI and Falter-stuff
+  - falter-common
   - luci-app-olsr
   - luci-app-olsr-services
   - luci-mod-falter
-  - falter-common
   - luci-app-falter-owm
   - luci-app-falter-owm-ant
   - luci-app-falter-owm-cmd
   - luci-app-falter-owm-gui
+  
+  # utils
+  - tcpdump-mini
+  - mtr
+  - ip
```

Comment:
Do you mean ip-full, which is also included fore role_uplink_gateway?

Perhaps it should be shipped via group_vars/all/..

.. If so, then please remove it from role_uplink_gateways list.